logging/testing/util.go

```diff
@@ -27,13 +27,19 @@ import (
 )
 
 // TestLogger gets a logger to use in unit and end to end tests
-func TestLogger(t *testing.T) *zap.SugaredLogger {
+func TestLogger(i interface{}) *zap.SugaredLogger {
```

Comment:
You can actually just change this to `t zaptest.TestingT`. No need for type assertions then :).

Reply:
Can we add this to `TestContextWithLogger` for completeness as well?

Reply:
Sure 
Thank you :+1: